contrib/src/main/java/com/datatorrent/contrib/enrich/AbstractEnricher.java

```diff
@@ -185,13 +185,18 @@ public void activate(Context context)
     try {
       cacheManager.initialize();
     } catch (IOException e) {
-      throw new RuntimeException("Unable to initialize primary cache", e);
+      throw new RuntimeException("Unable to initialize cache store", e);
```

Comment:
Why would be just cache store initialization failure, the backup could fail as well, isn't it. Shouldn't this be unable to initialize cache manager.